index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Microverse Survey Form</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <h1 id="title">Survey Form</h1>
+    <p id="description">This is a survey form by Kalumuna and Kehinde</p>
+
+    <form id="survey-form">
+      <label for="name" id="name-label">Name</label>
+      <input type="text" id="name" name="name" placeholder="Enter your name">
+      <label for="email" id="email-label">Email</label>
+      <input type="text" id="email" name="email" placeholder="Enter your email address">
+      <label for="name" id="number-label">Age</label>
+      <input type="text" id="number" name="number" min="18" max="40"placeholder="Enter your age">
```

Comment:
Please give `type` of `number` to the "Age" input

Reply:
- [ ] Please always try to explain the reasoning for your suggestion (  what are the advantages of the type attribute ). it may not be obvious to the reader why you are making this suggestion.

> For example, you can say
I think it would be better to add the attribute `type="number"` to the age input to prevent the user from entering any other type. The type is used to determine what the input will look like and gives a basic validation to make sure the user types in correct.

Reply:
Thanks for the suggestion. Duly noted.